tempo/auth/README.md

```diff
@@ -0,0 +1,73 @@
+# Tempo authentication
+
+This is an example about how to deploy a Tempo instance using the Tempo Operator. It receives traces from two tenants (`prod` and `dev`).
+
+
+The Tempo deployment accepts traces from two tenants: `dev` and `prod`. They way to identify the tenants is through the `X-Scope-OrgID` OTLP header. When the `X-Scope-OrgID` header is set to `dev` in the trace, the tenant is `dev`. When the `X-Scope-OrgID` header is set to `prod` in the trace, the tenant is `prod`.
+
+It also enables the creation of the Jaeger UI `.spec.template.queryFrontend.jaegerQuery.enabled` and the gateway (`.spec.template.gateway.enabled`). You can get the URL to the UI for a given tenant with the following command:
+
+```sh
+echo https://$(oc get routes tempo-authentication-example-gateway -o jsonpath='{.spec.host}')/<tenant name>
+```
+
+For instance, for the `dev` tenant, it would be:
+```sh
+echo https://$(oc get routes tempo-authentication-example-gateway -o jsonpath='{.spec.host}')/dev
+```
+
+The tenant needs permissions to the `tempo.grafana.com` API Group to write traces. You can follow this example to know
+how to provide write permissions to the `tenant-sa` `ServiceAccount` to write traces for the `dev` tenant:
```

Comment:
I wonder if the example should also describe how to use the `tenant-sa` account? Either deploying an otel collector or running a trace generate using this service account?

It's not absolutely required, but I think it'd be good, wdyt?

Reply:
That is something I'll do as part of #12, where both examples will be "crosslinked"

Reply:
ok, sounds good!

Reply:
Do you mean for the name of the service account or for what?